R/set_spark_home_env_var.R

```diff
@@ -0,0 +1,27 @@
+#' Set the SPARK_HOME environment variable
+#' 
+#' Set the \code{SPARK_HOME} environment variable. This slightly speeds up some 
+#' operations, including the connection time.
+#' @param path A string containing the path to the installation location of
+#' Spark. If \code{NULL}, the path to the most latest Spark/Hadoop versions is
+#' used. 
+#' @param verbose Logical. Should the function explain what is it doing?
+#' @return The function is mostly invoked for the side-effect of setting the 
+#' \code{SPARK_HOME} environment variable. It also returns \code{TRUE} if the 
+#' environment was successfully set, nad \code{FALSE} otherwise.
+#' @examples
+#' \dontrun{
+#' # Not run due to side-effects
+#' set_spark_home_env_var()
+#' }
+#' @export
+set_spark_home_env_var <- function(path = NULL, verbose = is.null(path)) {
```

Comment:
I've moved `spark_home_dir()` to `spark_home.R` via https://github.com/rstudio/sparklyr/pull/575/commits/29f10d109a20f3f744401d4a0e9f5d155a69edda.